README.md

```diff
@@ -29,22 +34,29 @@ length limit is based on a survey of the defaults for various JSON parsers.
 extension which MUST reference all the parameters.
 * Type: Data type for the parameter. In order to allow validation and limit the scope for implementation specific variations,
 parameters are typed.
-  * integer - may be signed or not as specified in the range parameter.
+  * integer - may be signed or not as specified in the range property.
   * string - aligned with JSON strings, these should be UTF-8 encoded and avoid control characters.  
   * enumerated - one of an ordered set of labels which MUST conform to the same limitations as parameter names.  
-  * boolean - may have the values false or true (lower case and unquoted as in JSON)
-  * For integer parameters the range specifies minimum and maximum values, separated by a comma, which MUST be integers themselves.
-  * For string parameters, the range specifies the maximum length of the string as an integer number of characters, not bytes. Again, based on a survey of parsers, try to keep strings shorter than 4095 characters.
+  * boolean - MUST have the value *false* or *true* (lower case and unquoted as in JSON)
+* Range: Further qualifies the valid values for a parameter depending on its type.
+  * For integer parameters, the range specifies minimum and maximum values, separated by a comma, which MUST be integers themselves.
+  * For string parameters, the range specifies the maximum length of the string as an integer number of characters, not bytes. Again, based on a survey of parsers, strings SHOULD be shorter than 4095 characters.
```

Comment:
I would expect the `range` for string parameters to somehow define the allowable values. Would a regex or enumerated list make sense? I am not sure I see the benefit of using the `range` property to define string lengths.

Reply:
Park this for a later pull request.

Reply:
@awoods to create an issue for discussion